src/main/scala/play/modules/reactivemongo/MongoController.scala

```diff
@@ -124,23 +124,29 @@ trait MongoController {
 
   /** Gets a body parser that will save a file sent with multipart/form-data into the given GridFS store. */
   def gridFSBodyParser(gfs: GridFS[JSONSerializationPack.type])(implicit readFileReader: Reads[ReadFile[JSONSerializationPack.type, JsValue]], ec: ExecutionContext): BodyParser[MultipartFormData[Future[ReadFile[JSONSerializationPack.type, JsValue]]]] = {
```

Comment:
Maintaining binary compatibility, especially for such case, it's not part of the development constraints for this project. When there is a new version, people rebuild their Play app. As soon as the API is compatible, there is no issue.